src/main/resources/database/changes/release-29/case_action_addition.sql

```diff
@@ -0,0 +1,63 @@
+-- Adding required indexes for case_action view.
+Create index idx_casegroup_collection_exercise_id
+ON casesvc.casegroup USING btree (collection_exercise_id);
+Create index idx_case_active_enrolment
```

Comment:
there's a few places in this file, where they could be changed to uppercase

Reply:
👍